app/views/shared/_back_to_login.html.erb

```diff
@@ -0,0 +1,3 @@
+<div data-go-back-link='true' style='display:none;line-height:3'>
```

Comment:
I think the `display:none` is okay in a style tag when we want to be absolutely sure it's hidden immediately even before any css is loaded, but we should avoid styling things like `line-height` directly in the markup like this.